.pipelines/singletenancy/dualstack-overlay/dualstackoverlay-e2e-job-template.yaml

```diff
@@ -0,0 +1,63 @@
+parameters:
+  name: ""
+  displayName: ""
+  pipelineBuildImage: "$(BUILD_IMAGE)"
+  testDropgz: ""
+  clusterName: ""
+
+stages:
+  - stage: ${{ parameters.name }}
+    displayName: E2E - ${{ parameters.displayName }}
+    dependsOn: 
+    - setup
+    - publish
+    jobs:
+      - job: ${{ parameters.name }}
+        displayName: DualStack Overlay Test Suite - (${{ parameters.name }})
+        timeoutInMinutes: 120
```

Comment:
What's the current time it take to complete this stage ?

Reply:
around 30 mins

Reply:
We can delete the timeout as it defaults to 60 minutes.